client/app/domain/domain-validator.service.js

```diff
@@ -747,5 +748,24 @@ angular.module("services").service(
             }
             return "";
         }
+
+
+        /**
+         * Transform CAA target to expected value
+         * @param {object} target
+         * @returns {string}
+         */
+        static transformCAATarget (target) {
+            if (
```

Comment:
You should extract the condition in an external variable :

```javascript
const condition = a && b && c;
if (condition) {
    // ...
}
```

It would be more readable :smile: